pkg/model/logstore/logstore.go

```diff
@@ -585,6 +585,22 @@ func (s *LogStore) logHelper(spansToLog map[SpanID]*Span, showManifestPrefix boo
 			}
 		}
 
+		if segment.Fields[logger.FieldNameBuildEvent] == "0" {
```

Comment:
i think you're going to need to sync and rewrite this bit, because this rendering pipeline has changed a lot